decidim-meetings/app/models/decidim/meetings/meeting.rb

```diff
@@ -78,7 +87,27 @@ def users_to_notify_on_comment_created
       def can_participate?(user)
         return true unless participatory_space.try(:private_space?)
         return true if participatory_space.try(:private_space?) && participatory_space.users.include?(user)
-        return false if participatory_space.try(:private_space?) && participatory_space.try(:is_transparent?)
+        return false if participatory_space.try(:private_space?) && participatory_space.try(:transparent?)
+      end
+
+      def can_participate_meeting?(current_user)
```

Comment:
Why this needs to be a separate method from `can_participate?`

Reply:
Because Rubocop complains...

Reply:
I think you can add a skip rule to `rubocop.yml`